### PR TITLE
reduce booleans to true/false, do not allow nil

### PR DESCRIPTION
**Pull request by grosser**
solves #382

### CODE REVIEW

**Comment by grosser** [comment]
@aried3r

**Comment by aried3r** [comment]
Thanks!

File: lib/generators/templates/rpush_3_0_1_updates.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Rpush301Updates < ActiveRecord::VERSION::MAJOR >= 5 ? ActiveRecord::Migration[5.0] : ActiveRecord::Migration
+  def self.up
+    change_column_null :rpush_notifications, :mutable_content, false
+    change_column_null :rpush_notifications, :content_available, false
+    change_column_null :rapns_notifications, :alert_is_json, false
```
Comment (redross) [inline lib/generators/templates/rpush_3_0_1_updates.rb]: I'm most likely doing something wrong (late night coding :smile: ), but this part fails for me locally (the table `rapns_notifications` doesn't exist). Shouldn't it be:
```
change_column_null :rpush_notifications, :alert_is_json, false
```

Comment (grosser) [reply lib/generators/templates/rpush_3_0_1_updates.rb]: yes!

... table got renamed :(

https://github.com/rpush/rpush/blob/2fa09cc352eae4abff8be7e0eec6324786df6e7f/lib/generators/templates/rename_rapns_to_rpush.rb

Comment (grosser) [reply lib/generators/templates/rpush_3_0_1_updates.rb]: https://github.com/rpush/rpush/pull/396

**Pull request merged**